Frends.AzureBlobStorage.ListBlobsInContainer/Frends.AzureBlobStorage.ListBlobsInContainer/Definitions/Result.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+
+namespace Frends.AzureBlobStorage.ListBlobsInContainer.Definitions
```

Comment:
Just in general as with other comments - no need for indent, it would make it harder to write more complex docs.